content/games.html

```diff
@@ -78,7 +78,7 @@
           rowClass = 'greyout';
 
         var row = $('<tr class="'+rowClass+'">');
-        row.append('<td>'+game['name'].replace(/\\\'/g, "'")+'<br /><span class="gameSubline">'+game['address']+'</span></td>');
+        row.append('<td><a href="openra://'+game['address']+'">'+game['name'].replace(/\\\'/g, "'")+'</a><br /><span class="gameSubline">'+game['address']+'</span></td>');
```

Comment:
I think it would make more sense to linkify the ip address instead of the server name.